packages/flutter_test/lib/src/binding.dart

```diff
@@ -700,13 +747,41 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
     // The LiveTestWidgetsFlutterBinding overrides this to report the exception to the console.
   }
 
+  Future<void> _handleMessage(Object? mockMessage) async {
```

Comment:
nit: `_handleMessage` sounds too generic. This is handling announcement messages specifically. Maybe call it `_handleAnnouncementMessage`?

Reply:
Done!